src/lib/apparmor/profiles.rb

```diff
@@ -159,7 +159,9 @@ def redraw_table
     def changeMode_handler
       selected_item = Yast::UI.QueryWidget(Id(:entries_table), :CurrentItem)
       log.info "Toggling #{selected_item}"
-      @profiles.toggle(selected_item)
+      if selected_item != nil
+        @profiles.toggle(selected_item)
```

Comment:
you can write it as `@profiles.toggle(selected_item) if selected_item`

Also please add entry to changelog and increase version. Thanks